src/AddressAutoComplete.js

```diff
@@ -29,8 +77,48 @@ export default class AddressAutoComplete extends Component {
   }
 
   getNoResultsMessage() {
-    const { noResultsMessage } = this.props;
-    return noResultsMessage;
+    const { t } = this.props;
+    return t('addressAutoComplete.noResults');
+  }
+
+  getStatusSelectedOptionMessage(selectedOption) {
+    const { t } = this.props;
+    return t('addressAutoComplete.statusSelectedOption', {
+      option: selectedOption
+    });
+  }
+
+  getStatusNoResultsMessage() {
+    const { t } = this.props;
+    const statusNoResults = t('addressAutoComplete.statusNoResults');
+
+    // don't repeat "No matching addresses" over and over
+    if (this.currentStatusMessage === statusNoResults) {
+      return '';
+    }
+
+    this.currentStatusMessage = statusNoResults;
+    return statusNoResults;
+  }
+
+  getStatusResultsMessage(length, contentSelectedOption) {
+    const { t } = this.props;
+
+    if (!contentSelectedOption) {
```

Comment:
This shortcuts any messages like "5 matching addresses are available". My test version did the opposite and only spoke a message if there wasn't a selected option, and relied on `tStatusSelectedOption` for when an option was selected.